packages/components/file-size/stories/file-size.happo.js

```diff
@@ -1,35 +1,35 @@
-import { storiesOf } from '@open-wc/demoing-storybook';
-import '@tradeshift/elements';
-import '@tradeshift/elements.file-size';
-import { createHappoStories } from '../../../../.storybook-happo/utils';
-
-storiesOf('ts-file-size', module).add('test', () => {
-	const properties = {
-		size: { small: 12345, medium: 12345678, large: 9999 * 1234567 },
-		decimalPoint: {
-			zero: 0,
-			one: 1,
-			two: 2
-		},
-		variant: {
-			default: 'default',
-			title: 'title',
-			subtitle: 'subtitle'
-		},
-		color: {
-			default: 'default',
-			danger: 'danger',
-			primary: 'primary',
-			warning: 'warning',
-			success: 'success'
-		}
-	};
-
-	const options = {
-		columns: 4,
-		persistent_props: {
-			size: 123456789
-		}
-	};
-	return createHappoStories('file-size', properties, 'Button', options);
-});
+// import { storiesOf } from '@open-wc/demoing-storybook';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ACommentedCode)